lms/static/js/dashboard/legacy.js

```diff
@@ -224,6 +224,42 @@
              return false;
          });
 
+         $("#send_cta_email").click(function (e) {
+            console.log(urls.sendAccountActivationEmail);
+             $.ajax({
+                 type: 'POST',
+                 url: urls.sendAccountActivationEmail,
+                 data: $(this).serializeArray(),
+                 success: function(data) {
+                     if (data.success) {
+                         location.href = urls.dashboard;
```

Comment:
Why we are reloading the page for both success and error cases? Instead, can we just simply close the popup?